fastly/base_fastly_service_v1.go

```diff
@@ -74,7 +74,7 @@ func (h *DefaultServiceAttributeHandler) MustProcess(d *schema.ResourceData, ini
 
 type VCLLoggingAttributes struct {
 	format            string
-	formatVersion     uint
+	formatVersion     *uint
```

Comment:
`VCLLoggingAttributes` in each logging endpoint is populated regardless of the service type like:
```
		var vla = h.getVCLLoggingAttributes(sf)
		opts := gofastly.CreateGCSInput{
			Service:           d.Id(),
			Version:           latestVersion,
			Name:              sf["name"].(string),
			User:              sf["email"].(string),
			Bucket:            sf["bucket_name"].(string),
			SecretKey:         sf["secret_key"].(string),
			Path:              sf["path"].(string),
			Period:            uint(sf["period"].(int)),
			GzipLevel:         uint8(sf["gzip_level"].(int)),
			TimestampFormat:   sf["timestamp_format"].(string),
			MessageType:       sf["message_type"].(string),
			Format:            vla.format,
			ResponseCondition: vla.responseCondition,
			Placement:         vla.placement,
		}

```

This simplifies the logic when creating the Input object requests as it removes the need to have an `if`. It can cause problems with the defaults (which this change fixes). 
It will probably be a larger refactoring if we want to change the implementation and my goal here to fix the regression to get the acceptance tests back to green.